udata/core/dataset/rdf.py

```diff
@@ -331,6 +331,7 @@ def dataset_to_rdf(dataset: Dataset, graph: Optional[Graph] = None) -> RdfResour
     if frequency:
         d.set(DCT.accrualPeriodicity, frequency)
 
+    # FIXME: owner should map to rightsHolder instead of publisher?
```

Comment:
SEMIC maps [ISO-19139 "owner" to "DCT.rightsHolder"](https://github.com/SEMICeu/iso-19139-to-dcat-ap/blob/main/iso-19139-to-dcat-ap.xsl#L1953). We might want to do the same here?